docs/cody/clients/install-jetbrains.mdx

```diff
@@ -97,17 +93,59 @@ In addition, Autocomplete suggestions use the same color as inline parameter hin
 
 ## Commands
 
-Cody with JetBrains offers quick, ready-to-use Commands for common actions to write, describe, fix, and smell code. These allow you to run predefined actions with smart context-fetching anywhere in the editor, like:
+Cody with JetBrains offers quick, ready-to-use Commands for common actions to write, describe, fix, and smell code. These allow you to run predefined actions with smart context-fetching anywhere in the editor.
+
+These are broadly categorized into the Edit Commands and Chat Commands.
+
+- **Edit Code**: Makes inline code edits. You also get the option to select LLM for edit suggestions
+- **Document Code**: Create inline documentation for your code
+- **Generate Unit Test**: Creates inline unit tests for your code-
+- **Smell Code**: Finds code smells in your file
+- **Explain Code**: Expains code for your file
+
+![jetbrains-commands](https://storage.googleapis.com/sourcegraph-assets/Docs/jb-commands-new-ga.png)
 
-- Explain Selected Code
-- Generate Unit Test
-- Smell Code
+Let's learn about how to use some of these commands:
 
-![jetbrains-commands](https://storage.googleapis.com/sourcegraph-assets/Docs/jb-commands.png)
+### Inline code edits
+
+You can make edits to your code directly in your file without opening the chat window. The **Edit Code** command makes direct code edits, refactors, or bug fixes.
+
+You can run the inline edit command either on the selected code snippet or the entire file. To do so, press `Shift + Ctrl + Enter`. This opens a floating editor where you can describe the change you want to make.
+
+Once you enter your prompt, Cody will perform inline edits that you can **Accept**, **Undo**, or **Show diff** for the change. You can also click **Retry** to iterate your prompt and get alternate suggestions.
+
+<video width="1920" height="1080" loop playsInline controls style={{ width: '100%', height: 'auto' }}>
+  <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/inline-code-edits.mp4" type="video/mp4"/>
+</video>
+
+### Inline unit tests
+
+The Generate Unit Test command uses the code edit functionality to create unit tests. To generate unit tests for your code, type `Shift + Ctrl + T` or click the **Generate Unit Test** command in the Cody sidebar.
+
+When you run this command, it performs the following actions:
+
+- Check if you’re using a test framework
+- Check if you already have a test file created
+- Insert new unit tests directly in the existing test file or create a new file if none exists
+
+<video width="1920" height="1080" loop playsInline controls style={{ width: '100%', height: 'auto' }}>
+  <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/inline-unit-tests.mp4" type="video/mp4"/>
+</video>
+
+Alternatively, you can **Accept**, **Undo**, or **Show diff** for the change. You can also click **Retry** to iterate your prompt and get alternate suggestions.
```

Comment:
I think we can cut "Alternatively," here? I don't know what it means. Also, we've removed the "Edit & Retry" code lens from code that's generated by a Test command. (Same behavior in VSC.)